docs/the-contribution-workflow.md

```diff
@@ -0,0 +1,245 @@
+## Contributing to Exercism
+
+Open source contributions can be intimidating! As you're looking around, wondering where to dive in, you might think everyone _else_ knows what's going on in the code base, or has better grasp of the language, or seems to be well known around the project.
+
+As a total beginner, where do you begin?
+
+Right here. This guide will walk you through your first open source contribution.
+
+It's going to target the smallest possible fix (improving documentation), as a "bridge" to more complex contributions later.
+
+First, go take a look at [Contributing to Exercism.io](https://github.com/exercism/exercism.io/blob/master/CONTRIBUTING.md#submitting-code-changes)
+
+If not all of that makes sense to you, keep reading. This is for you.
+
+
+## Table of Contents
+
+- [Find the right contribution](#find-the-right-contribution)
+- [Claiming that contribution](#claiming-that-contribution)
+- [Working on the code](#working-on-the-code)
+- [Submitting your contribution](#submitting-your-contribution)
+
+# Find the right contribution
+
+As mentioned in the contribution guideline, take a look at the [open issues](https://github.com/exercism/exercism.io/issues). There's a lot here, so you might want to filter by `label:"good first patch"` [(like this)](https://github.com/exercism/exercism.io/labels/good%20first%20patch).
+
+Read through the conversations, and decide if it seems like a good fit.
+
+(Another good place to start: [Finding your way](https://github.com/exercism/docs/blob/master/finding-your-way.md))
+
+As an example, I (person who is writing this guide) went through the above steps and found [this conversation](https://github.com/exercism/exercism.io/issues/3252). I thought "I think I can do that", and decided I wanted to.
+
+# Claiming that contribution
+
+Why would you claim an issue?
+
+It will make sure that you're not duplicating someone else's work, or someone else doesn't start duplicating yours. It also helps the project maintainers know what is being worked on. It's hard to over communicate, so you should certainly claim an issue.
+
+Different projects have different ways of "claiming" an issue, but for Exercism, just drop a comment in there, [like so](https://github.com/exercism/exercism.io/issues/3252#issuecomment-312428804).
+
+If it makes sense, someone from the project will jump in, offer some guidance, and (generally) be pleased with your offer.
+
+# Working on the code
+
+OK, you've got an issue, you've got approval to work on it. Now's the fun part.
+
+Now you need to get Exercism running on your local machine.
```

Comment:
Ahhh. OK, so this bit is going to change a LOT and I have no idea what it's going to look like yet, unfortunately. I'll know more in a few weeks. If we go with a "tracks" guide, then that's easier to define now (go look at their readme, and if they have one, the contributing guide, for how to get the exercises running locally).